Argo/Functions/sequence.swift

```diff
@@ -1,11 +1,30 @@
 public func sequence<T>(xs: [Decoded<T>]) -> Decoded<[T]> {
-  return xs.reduce(pure([])) { accum, elem in
-    curry(+) <^> accum <*> ({ [$0] } <^> elem)
+  var accum: [T] = []
+  accum.reserveCapacity(xs.count)
+  
```

Comment:
Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)